src/effects/effectmanifest.h

```diff
@@ -87,6 +88,14 @@ class EffectManifest {
         m_isForFilterKnob = value;
     }
 
+    virtual const double& superKnobScaleStart() const {
+        return m_dSuperKnobScaleStart;
+    }
+
+    virtual void setSuperKnobScaleStart(const double value) {
+        m_dSuperKnobScaleStart = value;
```

Comment:
This will not always work, since the super knob mapping is changeable by the user. This featured is enabled in Shade (by clicking on the bar below the effect parameter knobs)